src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -103,28 +103,67 @@ const CancelCollectionButton = (props) => {
   });
 };
 
+const ResubmitCollectionButton = (props) => {
+  const { collection } = props;
+  return h(SimpleButton, {
+    keyProp: `resubmit-collection-${collection.id}`,
+    label: 'Revise',
+    //todo: should we filter out any statuses that could be resubmitted? is this the correct place for such logic?
+    //disabled: includes(determineCollectionStatus(collection))(['Under Election']),
+    baseColor: Theme.palette.secondary,
+    additionalStyle: {
+      width: '30%',
+      padding: '2%',
+      fontSize: '1.45rem',
+    },
+    onClick: () => props.showConfirmationModal(collection)
+  });
+};
+
 const actionsCellData = ({collection, showConfirmationModal}) => {
   const { darCollectionId } = collection;
   const cancelButtonTemplate = h(CancelCollectionButton, {
     collection,
     showConfirmationModal
   });
-
-  return {
-    isComponent: true,
-    id: darCollectionId,
-    label: 'cancel-button',
-    data: div(
-      {
-        style: {
-          display: 'flex',
-          justifyContent: 'left'
+  const resubmitButtonTemplate = h(ResubmitCollectionButton, {
+    collection,
+    showConfirmationModal
+  });
+  if(isCollectionCanceled(collection) === true)
+  {
```

Comment:
`isCollectionCanceled` already returns a boolean, so this can be simplified
```suggestion
  if (isCollectionCanceled(collection)) {
```